opaque-derive/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "opaque-derive"
+version = "0.1.0"
+repository = "https://github.com/novifinancial/opaque-ke"
+authors = ["Kevin Lewi <klewi@fb.com>", "François Garillot <fga@fb.com>"]
+license = "MIT"
+edition = "2018"
```

Comment:
Do we need all of these here?

Reply:
Well, now that we do have to publish opaque-derive, I'm afraid we do.